test/shared/src/main/scala/zio/test/DefaultTestRunner.scala

```diff
@@ -1,27 +0,0 @@
-/*
- * Copyright 2019 John A. De Goes and the ZIO Contributors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package zio.test
-
-import zio.test.environment._
-
-/**
- * A `Runner` that provides a default testable environment.
- */
-object DefaultTestRunner
```

Comment:
Let's move this to the ZIO Test package object as `val defaultTestRunner = TestRunner(TestExecutor.managed(testEnvironmentManaged))` and then we can refer to it in `DefaultRunnableSpec`.